Firmware/power_panic.cpp

```diff
@@ -191,13 +191,26 @@ void uvlo_() {
     eeprom_update_float_notify((float*)(EEPROM_UVLO_LA_K), extruder_advance_K);
 #endif
 
+#ifdef PREVENT_DANGEROUS_EXTRUDE
+    // Save min extrude temp
+    eeprom_update_word_notify((uint16_t*)EEPROM_UVLO_EXTRUDE_MINTEMP, extrude_min_temp);
+#endif //PREVENT_DANGEROUS_EXTRUDE
+    // Save max acceleration mm per s2
+    eeprom_update_block_notify(cs.max_acceleration_mm_per_s2_normal, (uint32_t *)EEPROM_UVLO_ACCELL_MM_S2_NORMAL, sizeof(cs.max_acceleration_mm_per_s2_normal));
+    eeprom_update_block_notify(cs.max_acceleration_mm_per_s2_silent, (uint32_t *)EEPROM_UVLO_ACCELL_MM_S2_SILENT, sizeof(cs.max_acceleration_mm_per_s2_silent));
+    // Save feedrates
+    eeprom_update_block_notify(cs.max_feedrate_normal, (float *)EEPROM_UVLO_MAX_FEEDRATE_NORMAL, sizeof(cs.max_feedrate_normal));
+    eeprom_update_block_notify(cs.max_feedrate_silent, (float *)EEPROM_UVLO_MAX_FEEDRATE_SILENT, sizeof(cs.max_feedrate_silent));
+    // Save Jerk
```

Comment:
Just my opinion, but I feel these comments aren't useful:

```
// Save min extrude temp
// Save max acceleration mm per s2
// Save feedrates
// Save Jerk
```

I would remove them as the variable names themselves tell me the same information